react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+function RegistrationForm() {
+  const [userName, setUserName] = useState('');
+  const [pw, setPw] = useState('');
+
+  function handleUsernameChange(event) {
+    setUserName(event.target.value);
+  }
+  function handlePasswordChange(event) {
+    setPw(event.target.value);
+  }
+  function handleSubmit(event) {
+    event.preventDefault();
+    if (!userName || !pw) {
+      alert('Please input username and passsword');
+    }
+    if (userName && pw) {
```

Comment:
This check isn't needed since your check above has covered this case. The easiest way to handle this is to either return after the check above (at line 17) or to use an `else`.